pubspec.yaml

```diff
@@ -7,12 +7,12 @@ environment:
   sdk: ">=3.2.3 <4.0.0"
 
 dependencies:
-  auto_route: ^7.8.4
+  auto_route: ^8.0.3
```

Comment:
I'll have to revert this upgrade so others don't get problems when working on the project.